workflows/raredisease.nf

```diff
@@ -560,6 +558,31 @@ workflow.onComplete {
     }
 }
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    FUNCTIONS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+def make_ped(samples) {
+
+    def case_name  = samples[0].case_id
+    def outfile  = workDir.resolve("$case_name" + '.ped')
+    outfile.text = ['#family_id', 'sample_id', 'father', 'mother', 'sex', 'phenotype'].join('\t')
+    def samples_list = []
+    for(int i = 0; i<samples.size(); i++) {
+        sample_tokenized   =  samples[i].id.tokenize("_")
+        sample_tokenized.removeLast()
```

Comment:
Do we always have a `_T` file ending even in hte case of a single sample-lane input?